addons/web/static/tests/views/calendar/calendar_multi_create.test.js

```diff
@@ -14,6 +14,46 @@ import {
 
 import { CalendarModel } from "@web/views/calendar/calendar_model";
 
+class SimpleEvent extends models.Model {
+    name = fields.Char();
+    date_start = fields.Date();
+
+    _records = [
+        {
+            id: 1,
+            name: "simple event 1",
+            date_start: "2019-03-11",
+        },
+        {
+            id: 2,
+            name: "simple event 2",
+            date_start: "2019-03-12",
+        },
+        {
+            id: 3,
+            name: "simple event 3",
+            date_start: "2019-03-13",
+        },
+    ];
+
+    _views = {
+        calendar: `
+            <calendar date_start="date_start" scales="month" multi_create_view="multi_create_form" aggregate="id:count">
```

Comment:
I would define the view in the test directly, e.g.
```js
await mountView({
    arch: ...
    ...
});
```
It's always convenient to see the arch when reading the test